typescript/packages/subsurface-viewer/src/storybook/layers/WellsLayer.stories.tsx

```diff
@@ -865,18 +861,30 @@ const CoarseWellFactorComponent: React.FC<CoarseWellFactorProps> = (
     };
 
     return (
-        <SubsurfaceViewer {...propsWithLayers}>
-            {
-                // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-                /* @ts-expect-error */
-                <View id="view_1">
-                    <ViewFooter>
-                        Coarsen wells using &quot;dataTransform&quot; property.
-                        Adjustable factor typical range: [0.001, 1000]
-                    </ViewFooter>
-                </View>
-            }
-        </SubsurfaceViewer>
+        <Root>
+            <div className={classes.main}>
+                <SubsurfaceViewer {...propsWithLayers}>
+                    {
+                        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                        /* @ts-expect-error */
+                        <View id="view_1">
+                            <ViewFooter>
+                                Coarsen wells using &quot;dataTransform&quot;
+                                property.
+                            </ViewFooter>
+                        </View>
+                    }
+                </SubsurfaceViewer>
+            </div>
+            <Slider
+                min={0.1}
+                max={70}
+                defaultValue={0.1}
+                step={0.1}
+                onChange={handleChange}
+                valueLabelDisplay={"auto"}
+            />
```

Comment:
I'd suggest using [`argTypes`](https://storybook.js.org/docs/api/arg-types) on the story parameters instead. You can override the default control to a [`range`](https://storybook.js.org/docs/api/arg-types#controltype).